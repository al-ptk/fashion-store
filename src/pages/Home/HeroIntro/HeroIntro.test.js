import React from 'react';
import { render } from '@testing-library/react';
import '@testing-library/jest-dom';

// describe('Hero intro', () => {
//   it('renders all components', () => {
//     const { container } = render(<HeroIntro />);
//     console.log(container);
//     expect(container).toMatchSnapshot();
//   });
// });
