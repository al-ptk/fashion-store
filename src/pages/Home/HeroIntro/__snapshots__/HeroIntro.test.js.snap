// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero intro renders all components 1`] = `
<div>
  <section
    class="sc-bczRLJ"
  />
</div>
`;
